lib/AST/ModuleNameLookup.cpp

```diff
@@ -40,6 +43,8 @@ namespace {
 
 /// Returns true if this particular ValueDecl is overloadable.
 static bool isOverloadable(const ValueDecl *VD) {
+  // FIXME: This is very suspect; it doesn't really match how the rest of the
+  // compiler works anymore. How do we want cross-module shadowing to work?
```

Comment:
Would it be possible to point to a document or some code that shows how the rest of the compiler does work? Or say something about what is meant?